src/services/completions.ts

```diff
@@ -793,6 +793,7 @@ namespace ts.Completions {
             }
             ({ insertText, isSnippet, importAdder, sourceDisplay } = entry);
             source = CompletionSource.ObjectLiteralMethodSnippet;
+            sortText = (parseInt(sortText, 10) + 1).toString() as SortText;

```

Comment:
I guess with this approach I would just use `SortText.OptionalMember` directly. I had originally suggested string addition (`sortText + "0"`) because `"110"` would fall between `"11"` and `"12"`. Although I’m now realizing that what I had in my head was like

```ts
const obj: { method1(): void, method2(): void } = {
    m|
}/*  ^ _____________________
       | method1           |
       | method1(): void { |
       | method2           |
       | method2(): void { |
        -------------------
*/
```

and none of these would produce that 😕. It would be really helpful if we could assign the same sortText to all of these and have VS Code respect the original order.

Reply:
yeah, with so much going on regarding sorting the entries on the vscode side, I don't know how we could achieve this...